packages/react/error-boundary/src/ErrorBoundary.tsx

```diff
@@ -51,7 +51,6 @@ const initialState: ErrorBoundaryState = {
 
 class BaseErrorBoundary extends Component<ErrorBoundaryProps, ErrorBoundaryState> {
   state = initialState;
-  updatedWithError = false;
```

Comment:
I thought this updatedWithError is unnecessary field, there is no updatedWithError in https://github.com/bvaughn/react-error-boundary/blob/master/src/ErrorBoundary.ts too.

If this is matter, let me know why meaningful please.